server/auth/index.js

```diff
@@ -32,6 +32,18 @@ router.post('/signup', async (req, res, next) => {
   }
 })
 
+router.put('/edit/:id', async (req, res, next) => {
+  try {
+    await User.update(req.body, {
```

Comment:
See if you can get the info back from this first database call, instead of making two calls.